### PR TITLE
sad trombone for clown pda

### PR DESCRIPTION
**Pull request by dannno**
iunno

### CODE REVIEW

**Comment by Chocobro** [comment]
:+1:
should the clown's pda be changed to have a soundboard inside it with a whole bunch of comical sounds
should the clown's shoes have adjustable sounds
these are true questions for the ages

**Comment by WJohn** [comment]
:+1: x :100:

File: code/game/objects/items/devices/PDA/PDA.dm [hunk 1]
```diff
@@ -28,6 +28,7 @@ var/global/list/obj/item/device/pda/PDAs = list()
 	var/tnote = null //Current Texts
 	var/last_text //No text spamming
 	var/last_honk //Also no honk spamming that's bad too
+	var/last_trombone //yada yada spam
```
Comment (Cheridan) [inline code/game/objects/items/devices/PDA/PDA.dm]: Uh, why not use whatever spam var that the honk sound effect uses instead of making a new one?

Comment (dannno) [inline code/game/objects/items/devices/PDA/PDA.dm]: durrrrrrrrrrrrrrrrrrrr

**Comment by dannno** [comment]
fixed

**Comment by dannno** [comment]
conflicts killed

**Comment by dannno** [comment]
oh duh im a stupid

**Pull request merged**